### ISSUE TITLE
12d1 is missing

### ISSUE THREAD

**Issue by jfly** [thewca/wca-regulations#24]
This is causing 12d1a and 12d1b to be misindented in the markdown source.

Here's what's currently in wca-regulations-2013.md:

``` markdown
- 12d) Notation for Megaminx (scrambling notation only):
    - 12d1a) Clockwise, 72 degrees: U (upper face).
    - 12d1b) Anti-clockwise, 72 degrees: U' (upper face).
```

I think we want something like this instead:

``` markdown
- 12d) Notation for Megaminx (scrambling notation only):
  - 12d1)
    - 12d1a) Clockwise, 72 degrees: U (upper face).
    - 12d1b) Anti-clockwise, 72 degrees: U' (upper face).
```

**Comment by iVinshe** [comment]
Good catch, Jeremy.

On Monday, February 11, 2013, Jeremy Fleischman wrote:

> This is causing 12d1a and 12d1b to be misindented in the markdown source.
> 
> Here's what's currently in wca-regulations-2013.md:
> - 12d) Notation for Megaminx (scrambling notation only):
>   - 12d1a) Clockwise, 72 degrees: U (upper face).
>   - 12d1b) Anti-clockwise, 72 degrees: U' (upper face).
> 
> I think we want something like this instead:
> - 12d) Notation for Megaminx (scrambling notation only):
>   - 12d1)
>     - 12d1a) Clockwise, 72 degrees: U (upper face).
>     - 12d1b) Anti-clockwise, 72 degrees: U' (upper face).
>   
>   —
>   Reply to this email directly or view it on GitHubhttps://github.com/cubing/wca-documents/issues/24..

## 

Vincent Sheu
University of California, Berkeley
vsheu@berkeley.edu

**Comment by lgarron** [comment]
Fixed in b3133530e1969464c54b35420080aa038a185bec.
